src/pydata_sphinx_theme/theme/pydata_sphinx_theme/_templates/version-switcher.html

```diff
@@ -72,6 +72,12 @@
             if (entry.version == "{{ theme_switcher.get('version_match') }}") {
                 node.classList.add("active");
                 $("#version_switcher_button").text(entry.name);
+                // Add extra class to the button for the selected entry
+                if ("extra_classes" in entry) {
+                    $.each(entry.extra_classes, function(index, extra_class) {
+                      $("#version_switcher_menu").classList.add(extra_class);
+                    });
```

Comment:
```suggestion
                    $("#version_switcher_menu").classList.add(...entry.extra_classes);
```

Reply:
I think this accomplishes the same thing.

Reply:
This is some JS magic 😅 Thanks!